restapi/api_client.go

```diff
@@ -24,6 +24,7 @@ type apiClientOpt struct {
 	insecure            bool
 	username            string
 	password            string
+	bearer 				string
```

Comment:
Can you fix the formatting? Between bearer and string -> should be spaces instead of tabs. This goes for all other instances as well.

Reply:
Ran go linter, these issues should be fixed now